docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W
+# SPHINXOPTS    = -W
```

Comment:
```suggestion
SPHINXOPTS    = -W
```

Reply:
Don't worry, that's in a `TMP: ...` commit. Just turned off while I applied the wrecking ball..